src/NexusMods.DataModel/Games/GameCapabilities/FolderMatchInstallerCapability/IModInstallDestination.cs

```diff
@@ -0,0 +1,100 @@
+using NexusMods.Paths;
+
+namespace NexusMods.DataModel.Games.GameCapabilities.FolderMatchInstallerCapability;
+
+/// <summary>
+/// Represents a target used for where user mods may be manually installed when using human assisted installers
+/// such as the Advanced Installer.
+/// </summary>
+public interface IModInstallDestination
+{
+    /// <summary>
+    /// GamePath to which the relative mod file paths should appended to.
+    /// </summary>
+    public GamePath DestinationGamePath { get; }
+
+    /// <summary>
+    /// List of known recognizable file extensions for direct children of the target <see cref="DestinationGamePath"/>.
+    /// NOTE: Only include file extensions that are only likely to appear at this level of the folder hierarchy.
+    /// </summary>
+    public IEnumerable<Extension> KnownValidFileExtensions { get; }
+
+    /// <summary>
+    /// List of file extensions to discard when installing to this target.
+    /// </summary>
+    public IEnumerable<Extension> FileExtensionsToDiscard { get; }
```

Comment:
Do you intend to use this in the Manual Installer? 
I don't think it would be a good idea. 
It might still be useful for some other installers though.

Reply:
It's a leftover from when refined suggestions and basic suggestions were 1 ADR (they are now split).

More specifically leftover from: 
- https://github.com/Nexus-Mods/NexusMods.App/issues/634

(Which requires feedback from design about how the filtered data is used)

I can probably remove this as it's not currently needed until #634. When I was writing the code, I just copy-pasted it from the ADR.

Reply:
I'd make things simpler for now as we aren't sure if and how #634 will take place, but up to you

Reply:
I might as well leave it out for now, it's a copy paste job as this is already in ADR.

I'll fix that up when I get to my PC.